### ISSUE TITLE
Scroll issue iOS / Android within browser (Safari / Chrome)

### ISSUE THREAD

**Issue by guldi011** [nextcloud/deck#2105]
**Describe the bug**
Use of the deck app within the browser under iOS and Android is not possible (especially under iOS as there is no specific app) due to the browser scroll function (side wards) is blocked. Under iOS an additional failure occures:

Android:
- sidwards scrolling (up and down scrolling not yet tested) is possible as long as the finger is in touch with the display
- as soon as the finger is lifted from the display, no scrolling is possible at all and no list moving either, the GUI doesn't react at all to scroll or move gestures.
- to be able to scroll again, a card needs to be clicked ==> cart details pop-up on the right side ==> scrolling is again possibe, but again only once ==> procedure repeats

iOS:
- After selecting and loading a board, the icon with three lines is only half visible but can not be clicked. After trying to click it, it disapears completely ==> the first board list is covered by the "All boards" task list on the left side ==> the board is not usable
- the scrolling bahaviour failure is identical to the one described above under Android 

**To Reproduce**
1. see steps above

**Expected behavior**
- Scrolling possible under iOS and Android as expected (no intermediate selecting of cards required).
- "Three line" icon under iOS visible ==> left task list can be hidden ==> first list is accesible

**Client details:**
- iOS 13.5.1, iPad 9.7", safari
- Samsung Galaxy S7, chrome
- Samsung Tab A, chrome

**Comment by stefan-niedermann** [comment]
Also reproducable in Firefox on Android, so i guess it's an implementation mistake, no browser specific implementation detail.

**Comment by guldi011** [comment]
**Describe the bug**
Addendum: up- and downwards scrolling is affected identical as described above. I only tested this under Android / Chrome however.

btw:
I'm very (!) positive surprised by this cool Nextcloud app. This also includes the Android app ==> applaus to Niedermann IT (when does an iOS follows :-) even if I'm not an Apple disciple, some are).
This statement with the background as a pretty excessive JIRA user (large dev projects with several thousand tasks ("cards").

**Comment by juliushaertl** [comment]
Closing as a duplicate of of https://github.com/nextcloud/deck/issues/1960
